internal/deploy/deploy.go

```diff
@@ -58,7 +58,7 @@ func applyDNSThroughSdk(newIP string) {
 // moneycolfrontend, moneycolserver, elasticsearch, traefik, traefik-dev, collections-api
 func Deploy(appName string, environment string, propertiesMap map[string]string, dryRun bool) {
 	helmSetParams := make(map[string]string)
-	if appName == "traefik" || appName == "traefik-dev" {
+	if appName == "traefik" || appName == "traefik-dev" || appName == "test" {
```

Comment:
All this moneycol stuff I need to move away from here, could you please create issues for this and assign them to me?